GitHubReferences.ps1

```diff
@@ -0,0 +1,219 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+function Get-GitHubReference
+{
+<#
+    .SYNOPSIS
+        Retrieve a reference of a given GitHub repository.
+
+    .DESCRIPTION
+        Retrieve a reference of a given GitHub repository.
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER OwnerName
+        Owner of the repository.
+        If not supplied here, the DefaultOwnerName configuration property value will be used.
+
+    .PARAMETER RepositoryName
+        Name of the repository.
+        If not supplied here, the DefaultRepositoryName configuration property value will be used.
+
+    .PARAMETER Uri
+        Uri for the repository.
+        The OwnerName and RepositoryName will be extracted from here instead of needing to provide
+        them individually.
+
+    .PARAMETER Reference
+        Name of the reference, for example: "heads/<branch name>" for branches and "tags/<tag name>" for tags
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .PARAMETER NoStatus
+        If this switch is specified, long-running commands will run on the main thread
+        with no commandline status update.  When not specified, those commands run in
+        the background, enabling the command prompt to provide status information.
+        If not supplied here, the DefaultNoStatus configuration property value will be used.
+
+    .EXAMPLE
+        Get-GitHubReference -OwnerName Powershell -RepositoryName PowerShellForGitHub -Reference heads/master

```

Comment:
Since the time that your PR started, this repo has moved from the `PowerShell` org to the `microsoft` org...

```suggestion
        Get-GitHubReference -OwnerName microsoft -RepositoryName PowerShellForGitHub -Reference heads/master
```

Reply:
Will fix this in the other PR